docs/appending_events.md

```diff
@@ -62,8 +60,8 @@ EventStoreClient::DeserializedEvent.new(
   type: 'some-event-name',
   # Event data. Optional. Will default to `{}` (empty hash) if omitted
   data: { foo: :bar },
-  # Optional. Defaults to `{ 'type' => event_name_you_provided, 'content-type' => 'application/json' }`
```

Comment:
I removed `#metadata` from documentation at all, because user can't change there anything.
Example:

```ruby
EventStoreClient::DeserializedEvent.new(type: 'some-event', metadata: { 'type' => 'some-another-event', 'content-type' => 'text/html' })
# =>  #<EventStoreClient::DeserializedEvent:0x00007fd8b25abcb0 @data={}, @type="some-event", @metadata={"type"=>"some-event", "content-type"=>"application/json"}, @custom_metadata={}, @stream_name=nil, @stream_revision=nil, @prepare_position=nil, @commit_position=nil, @title=nil, @id=nil>
```